src/interfaces/curve/ICurvePool.sol

```diff
@@ -3,7 +3,10 @@ pragma solidity =0.8.16;
 
 // See https://curve.readthedocs.io/exchange-deposits.html#curve-stableswap-exchange-deposit-contracts
 /*  solhint-disable func-name-mixedcase, var-name-mixedcase */
-interface ICurvePool {
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface ICurvePool is IERC20 {
```

Comment:
The curve pool and the ERC20 are actually separate in the convex strategy. Let's keep them separate.